src/content/docs/public-pool/selfhost.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Self-Hosting Public Pool
+---
+
+1. 💻 [Windows](#-windows) - TODO
+2. 🍏 [MacOS](#-macos) - TODO
+3. 🐧 [Linux](#-linux)
+
+## 💻 Windows
+
+## 🍏 MacOS
+
+## 🐧 Linux
+
+### Installing Bitcoin Core
+
+- download latest release from https://bitcoin.org/en/download
+  TODO: note about knots and btcd being usable as well
+- extract everything in `bitcoin-XX.X/bin` to `/usr/local/bin`, and everything in `bitcoin-XX.X/share` to `/usr/local/share`
+
+### Installing NodeJS
+
+- ubuntu/debian: select the node.js runtime, your distro, and the latest lts version from https://nodesource.com/products/distributions
+- arch: use nvm to install the latest lts
+
+### Configuring Bitcoin
+
+#### Bitcoin Core/Knots
+
+MAYBE: have a community mining config?
```

Comment:
this would be a nice to have, but the config would just be boosting the mempool to a gig and the orphan tx count to 1k

```ini
dbcache=4000
maxorphantx=1000
maxmempool=1024
blockreconstructionextratxn=1000
```